build.gradle

```diff
@@ -20,7 +19,8 @@ allprojects {
     repositories {
         google()
         jcenter()
-        
+        maven { url 'https://jitpack.io' }
```

Comment:
what's that?

Reply:
That's for the calendar view, the one from Material was not made to customize days display and selection (it was too hard to try and change the display (which btw I realized later that the picture I send on Telegram was the basic calendar view of android ^^'))